library/x509_crt.c

```diff
@@ -2754,8 +2754,8 @@ static int x509_inet_pton_ipv6(const char *src, void *dst)
             p++;
```

Comment:
Since you're in this function, would you mind adding a blank line after `uint16_t addr[8];` to separate the variable declarations from the code body?

Reply:
Sorry, not worth iterating for this.